deploy/cluster-monitoring-config/fedramp/config.yaml

```diff
@@ -0,0 +1,9 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
```

Comment:
Think we're missing an expression here to only target FedRAMP clusters, since the other config still exists for Non-FR clusters